File: JAVA/Product.java
/* Saya Rifa Sania NIM 2206697 mengerjakan Latihan Praktikum 2 dalam mata kuliah DPBO
untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin... */

class Product{
    private String idProduct;
    private String name;
    private String brand;
    private String price;

    Product(){        
    }
    
    public String getIdProduct(){
        return this.idProduct;
    }

    public void setIdProduct(String idProduct){
        this.idProduct = idProduct;
    }

    public String getName(){
        return this.name;
    }

    public void setName(String name){
        this.name = name;
    }

    public String getBrand(){
        return this.brand;
    }

    public void setBrand(String brand){
        this.brand = brand;
    }

    public String getPrice(){
        return this.price;
    }

    public void setPrice(String price){
        this.price = price;
    }
}

File: PHP/Program/Product.php
<?php

class Product{
    private $idProduct;
    private $name;
    private $brand;
    private $price;

    function __construct(){
    }

    function setIdProduct($idProduct){
        $this->idProduct = $idProduct;
    }

    function getIdProduct(){
        return $this->idProduct;
    }

    function setName($name){
        $this->name = $name;
    }

    function getName(){
        return $this->name;
    }

    function setBrand($brand){
        $this->brand = $brand;
    }

    function getBrand(){
        return $this->brand;
    }

    function setPrice($price){
        $this->price = $price;
    }

    function getPrice(){
        return $this->price;
    }

    function __destruct(){
    }
}

?>